build

```diff
@@ -94,10 +94,13 @@ rm -f ${BINDIR}/stolon-keeper
 cp ${GOPATH}/bin/keeper ${BINDIR}/stolon-keeper
 
 # Copy binaries to Dockerfile image directories
-DOCKERIMAGE=${BASEDIR}/examples/kubernetes/image/docker
-
-mkdir -p ${DOCKERIMAGE}/bin/
-rm -f ${DOCKERIMAGE}/bin/*
-for cmd in stolon-keeper stolon-sentinel stolon-proxy stolonctl; do
-	cp ${BINDIR}/${cmd} ${DOCKERIMAGE}/bin/
+declare -a $DOCKERFILE_PATHS
```

Comment:
This should probably be `declare -a DOCKERFILE_PATHS` or it will be expanded to `declare -a` and just show the current bash arrays.